src/adapters/aws/api-gateway-v1.adapter.ts

```diff
@@ -122,7 +130,9 @@ export class ApiGatewayV1Adapter
    */
   public getRequest(event: APIGatewayProxyEvent): AdapterRequest {
     const method = event.httpMethod;
-    const headers = { ...event.headers };
+    const headers = !this.options?.lowercaseRequestHeaders
+      ? { ...event.headers }
+      : keysToLowercase(event.headers);
```

Comment:
Invert this comparison, just to be easier to read. is very easy to miss the `!` in the beginning.